settings/provisioner/openstack/openstack.spec

```diff
@@ -30,6 +30,12 @@ subparsers:
                       type: Value
                       help: An image id or name, on OpenStack cloud to provision the instance with. To see full list of images avaialable on the cloud, use 'glance image-list'.
                       required: yes
+            - title: ansible_ssh_user
```

Comment:
That's a bad name.

1. It's too long
1. User's don't care about ansible
1. It's an ansible keyword.

`--image-user` Is better